### DISCUSSION TITLE
Apollo federation with prisma

### DISCUSSION THREAD

**Discussion by gmencz** [prisma/prisma#2435] [Q&A] [answered]
I'd like to know if prisma supports apollo federation.

**Comment by ryands17** [comment, marked as answer]
Hey @gabrielmendezc :wave: 
Prisma 2 is a DB access layer that helps you query your database. It is independent of REST or GraphQL and so I wouldn't know that how would it be helpful in supporting Apollo Federation.

Could you provide a use-case that you have in mind for how you would like this to be supported in Prisma?

**Reply by gmencz** [reply]
You're right! My bad, thanks for clarifying.

**Comment by ludralph** [comment]
Hi @gmencz,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
